src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -247,6 +247,7 @@ public function get_items( $request ) {
 			'author_exclude' => 'author__not_in',
 			'exclude'        => 'post__not_in',
 			'include'        => 'post__in',
+			'ignore_sticky'  => 'ignore_sticky_posts',
```

Comment:
By using the value from `WP_Query` is this a little back-to-front? At the moment if the user does not want stickies to stick, the send `true`, if they do want them to stick they send `false`.

Should it be massaged for a positive name? (Genuine question, not a passive aggressive change request!)

Reply:
We could change it to `include_sticky`. Happy to address it as requested. I don't have a strong opinion here.

@spacedmonkey, @TimothyBJacobs, is there a naming convention guide for REST API parameters?

Reply:
Besides this naming dilemma, do you think this is good to merge?

I've been trying to come up with a better name, but having trouble there. Even `include_sticky` is too similar to the `include` argument.

The endpoint already supports the `sticky` argument, which is basically `post__in/post__not_in` for sticky posts.

Reply:
> Besides this naming dilemma, do you think this is good to merge?

Yes, I think it's good to merge putting aside the naming issue. Are you happy to give it a couple of days for others to provide feedback and if nothing comes in, commit it on Friday?

Reply:
Sounds good!